application/org.openjdk.jmc.ui.common/src/main/java/org/openjdk/jmc/ui/common/util/ThemeUtils.java

```diff
@@ -33,15 +34,17 @@ public static boolean isDarkTheme() {
 
 	private static final double BRIGHTNESS_THRESHOLD = 0.5;
 
-	private static boolean isDarkMode() {
+	private static void isDarkMode() {
```

Comment:
**nitpick:** I'd rename `isDarkMode` that better describe that it updates the state, e.g. 

```suggestion
	private static void computeCurrentThemeDarkModeStatus() {
```

Reply:
Done.